clients/upsmon.c

```diff
@@ -1270,21 +1272,32 @@ static int is_ups_critical(utype_t *ups)
 
 	if (ups->commstate == 0) {
 		/* Note: ECO mode not considered easily fatal here */
-		if (flag_isset(ups->status, ST_CAL)) {
+
+		/* FIXME: Consider equivalents of alarmcritical for over/trim/boost? */
+		if (flag_isset(ups->status, ST_OVER)) {
 			upslogx(LOG_WARNING,
-				"UPS [%s] was last known to be calibrating "
-				"and currently is not communicating, assuming dead",
+				"UPS [%s] was last known to be overloaded "
+				"and currently is not communicating, "
+				"but we make no assumptions if it is dead now",
 				ups->sys);
-			return 1;
 		}
```

Comment:
Hmm, I personally would have kept `OVER` for instant criticality on communications lost. Just not `TRIM`/`BOOST`. Because an overloaded UPS is - I think - a critical enough situation as it stands, and a possible fire hazard at that, so immediate shutdown seems warranted here (e.g. excess heat already melted the communication bus...)

Reply:
On the other hand, a super short spike in power draw (mass drive spin-up comes to mind here) may as well temporarily overwhelm the UPS and its communication abilities to the point of causing such a short communication blip and taking down the system... Not sure, perhaps we should make this a configuration toggle with `OVER` + nocomms defaulting to critical (for playing it safe)?

Reply:
Possibly, like with throttle variables for some other events/states.

Not sure I want to dive into this sort of change before the release though. As a newly reported situation we had not handled or reported before, this PR as is at least should not cause unexpected behaviors, I think - only more log noise when stars mis-align. Now just the more-frequent polling (like in an outage) remains, but I don't believe it would be a big problem per se.

Reply:
Fair, otherwise LGTM.